src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  display: grid;
+  grid-template-columns: $cardWidth;
```

Comment:
I think this property is redundant here 🤔

Reply:
You are right, I have moved it.
Thank you 👍